src/main/java/seedu/address/logic/commands/room/EditRoomCommand.java

```diff
@@ -60,8 +60,10 @@
     private final EditRoomDescriptor editRoomDescriptor;
 
     /**
-     * @param index              of the room in the filtered room list to edit
-     * @param editRoomDescriptor details to edit the room with
+     * Creates an EditRoomCommand to edit the room at the specified {@code Index}.
+     *
+     * @param index              of the room in the filtered room list to edit.
```

Comment:
Try not to use the parameter in the sentence.
Capitalise the sentence

Reply:
Fixed